modules/ipi-install-troubleshooting-misc-issues.adoc

```diff
@@ -0,0 +1,265 @@
+// Module included in the following assemblies:
+// //installing/installing_bare_metal_ipi/installing_bare_metal_ipi/ipi-install-troubleshooting.adoc
+
+[id="ipi-install-troubleshooting-misc-issues_{context}"]
+
+= Miscellaneous issues
+
+== Addressing the `runtime network not ready` error
+
+After the deployment of a cluster you may receive the following error:
+
+----
+`runtime network not ready: NetworkReady=false reason:NetworkPluginNotReady message:Network plugin returns error: Missing CNI default network`
+----
+
+The cluster network operator is responsible for deploying the networking components in response to a special object created by the installer. It runs very early in the installation process, after the Control Plane (master) nodes have come up but before the bootstrap control plane has been torn down. It can be indicative of more subtle installer issues, such as long delays in bringing up Control Plane (master) nodes or issues with `apiserver` communication.
+
+.Procedure
+
+. Inspect the pods in the `openshift-network-operator` namespace:
++
+[source,bash]
+----
+[kni@provisioner ~]$ oc get all -n openshift-network-operator
+----
++
+----
+NAME                                    READY STATUS            RESTARTS   AGE
+pod/network-operator-69dfd7b577-bg89v   0/1   ContainerCreating 0          149m
+----
+
+
+. On the `provisioner` node, determine that the network configuration exists:
++
+[source,bash]
+----
+[kni@provisioner ~]$ kubectl get network.config.openshift.io cluster -oyaml
+----
++
+----
+apiVersion: config.openshift.io/v1
+kind: Network
+metadata:
+  name: cluster
+spec:
+  serviceNetwork:
+  - 172.30.0.0/16
+  clusterNetwork:
+  - cidr: 10.128.0.0/14
+    hostPrefix: 23
+  networkType: OpenShiftSDN
+----
++
+If it does not exist, the installer did not create it. To determine why the installer did not create it, execute the following:
++
+[source,bash]
+----
+[kni@provisioner ~]$ openshift-install create manifests
+----
+
+. Check that the `network-operator` is running:
++
+[source,bash]
+----
+[kni@provisioner ~]$ kubectl -n openshift-network-operator get pods
+----
+
+. Retrieve the logs:
++
+[source,bash]
+----
+[kni@provisioner ~]$ kubectl -n openshift-network-operator logs -l "name=network-operator"
+----
++
+On high availability clusters with three or more Control Plane (master) nodes, the operator will perform leader election and all other operators will sleep. For additional details, see https://github.com/openshift/installer/blob/master/docs/user/troubleshooting.md[Troubleshooting].
+
+== Cluster nodes not getting the correct IPv6 address over DHCP
+
+If the cluster nodes are not getting the correct IPv6 address over DHCP, check the following:
```

Comment:
Should we spell out DHCP here, or is this a common enough acronymn?

Reply:
It's common enough. We spelled out when it's used in other compound acronyms like DUID, etc.